content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -42,12 +52,15 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-6857c8775f-zdv9r        2/2       Running   0           1m
 {{< /text >}}
 
-The number '0' in the 'RESTARTS' column means liveness probes worked fine. Readiness probes work in the same way and you can modify `liveness-command.yaml` accordingly to try it yourself.
+The number '0' in the 'RESTARTS' column means liveness probes worked fine.
+Readiness probes work in the same way and you can modify
+`liveness-command.yaml` accordingly to try it yourself.
 
 ### Mutual TLS enabled
 
-To enable mutual TLS for services in the default namespace, you must configure an authentication policy and a destination rule.
-Follow these steps to complete the configuration:
+To enable mutual TLS for services in the default namespace, you must
+configure an authentication policy and a destination rule. Follow these steps
+to complete the configuration:
```

Comment:
Please number all the steps in the procedure, not just the first.

Reply:
done